platform-controller/src/test/java/org/hobbit/controller/docker/ContainerManagerBasedTest.java

```diff
@@ -44,11 +45,20 @@ public void initManager() throws Exception {
 
     @After
     public void cleanUp() {
-        for (String taskId : containers) {
+        for (String containerId : containers) {
+            try {
+                dockerClient.stopContainer(containerId, 5);
+                dockerClient.removeContainer(containerId);
+            } catch (Exception e) {
+                LOGGER.warn("Couldn't cleanup container {}", containerId, e);
+            }
+        }
+        for (String taskId : tasks) {
```

Comment:
I am not sure about this. Wouldn't it be better to go through the tasks first and after that through the containers? What happens if the container that we stop and remove belongs to one of the tasks/services? Won't Docker try to recreate it? :thinking: